esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+        temperature time-series and one with the anomaly time-series
+
+  HadISST:
+    tier: 2
+    source: http://www.metoffice.gov.uk/hadobs/hadisst/data/download.html
+    last_access: 2019-02-08
+    info: |
+      Download and unzip "HadISST_ice.nc.gz" and "HadISST_sst.nc.gz".
+
+  HALOE:
+    tier: 2
+    source: Grooss, J.-U. and Russell III, J. M., Atmos. Chem. Phys., 5, 2797-2807, doi:10.5194/acp-5-2797-2005, 2005.
+    last_access: 2020-03-11
+    info: |
+      Download and processing instructions
+        Download and untar the supplementary material of the above reference.
+
+      Caveats
+        The time period includes incomplete years. According to the given
+        reference (Section 3): "Between October 1991 and August 2002, a total of
+        78600 HALOE profiles of O3, CH4, H2O, HCl, and HF mixing ratio were then
+        combined into bins of 5 degree equivalent latitude for 22 pressure levels
+        between 316 and 0.1 hPa and 12 months".
+        In order to create T3M fields a fake longitude coordinate is added.
+
+  HWSD:
+    tier: 3
+    source: https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1247
+    last_access: 2019-10-15
+    info: |
+      Download the following file:
+        HWSD_SOIL_CLM_RES.nc4
+      A registration is required
+
+  ISCCP-FH:
+    tier: 2
+    source: https://isccp.giss.nasa.gov/pub/flux-fh/tar-nc4_MPF/
+    last_access: 2019-11-07
+    info: |
+      1) Go to https://isccp.giss.nasa.gov/projects/flux.html and click on
+        "FH On-line Data"
+      2) username and password are provided on this website
+      3) go to directory tar-nc4_MPF/ (= monthly files)
+      3) download and unpack files
+
+  JMA-TRANSCOM:
+    tier: 3
+    source: http://www.globalcarbonatlas.org/en/content/atmospheric-inversions
+    last_access: 2019-07-02
+    info: |
+      To obtain the data sets it is necessary to contact Takashi Maki
+      (Department of Atmosphere, Ocean and Earth System Modeling Research,
+      Meteorological Research Institute, Tsukuba City, Japan). See link above
+      for more information.
+
+  LAI3g:
+    tier: 3
+    source: http://cliveg.bu.edu/modismisr/lai3g-fpar3g.html
+    last_access: 2019-05-03
+    info: |
+      To obtain the data sets it is necessary to contact Ranga B. Myneni
+      (Department of Earth and Environment, Boston University). See link above
+      for more information.
+
+      By default, this dataset is regridded to a 1°x1° grid (original resolution
+      is 1/12°). If you want to use the original resolution, remove the `regrid`
+      section in the configuration file (`LAI3g.yml`). Note that in this case,
+      preprocessing the dataset with ESMValTool (i.e. every time you run the
+      tool) can take a very long time (> 30 min).
+
+  LandFlux-EVAL:
+    tier: 3
+    source: https://data.iac.ethz.ch/landflux/
+    last_access: 2019-05-16
+    info: |
+      Download the following files:
```

Comment:
It would be good to include some instructions on how to use the automatic download procedure here as well (e.g. `APHRO-MA`) and give a generic example about using a .netrc file